requirements.in

```diff
@@ -1,12 +1,12 @@
 Sphinx==4.5.0
 backoff==2.0.1
-black==21.12b0
```

Comment:
just curious: does b0 stand for something?
also because this is a major version update

Reply:
didn't look too much but couldn't find anything in their docs

Reply:
Until very recently, they were doing "pre-release" releases and only just switched to doing regular releases. Python version conventions is covered in pep-440 (https://peps.python.org/pep-0440/).